src/stdune/fdecl.ml

```diff
@@ -0,0 +1,17 @@
+type 'a state =
+  | Unset
+  | Set of 'a
+
+type 'a t = { mutable state : 'a state }
```

Comment:
What do you think about adding a label that can be displayed in error messages?

Reply:
I suppose the backtrace should be enough

Reply:
Sure!